cpp/tests/strings/translate_tests.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/column/column.hpp>
+#include <cudf/column/column_factories.hpp>
+#include <cudf/strings/strings_column_view.hpp>
+#include <cudf/strings/translate.hpp>
+#include <cudf/utilities/error.hpp>
+
+#include <tests/utilities/base_fixture.hpp>
+#include <tests/utilities/column_wrapper.hpp>
+#include <tests/utilities/column_utilities.hpp>
+#include "./utilities.h"
+
+#include <vector>
+#include <gmock/gmock.h>
+
+
+struct StringsTranslateTest : public cudf::test::BaseFixture {};
+
+std::pair<cudf::char_utf8,cudf::char_utf8> make_entry( const char* from, const char* to )
+{
+    cudf::char_utf8 in, out = 0;
```

Comment:
Nope, it does not need to be initialized.

Reply:
In modern C++, we can do better than to write code where variables don't need to be initialized. If we were using a static analyzer, it would be complaining about this.

Reply:
Ok, no problem. I can initialize it to something.